### DISCUSSION TITLE
Glossary

### DISCUSSION THREAD

**Discussion by kerimovscreations** [kerimovscreations/ML-For-Beginners-AZ#18] [General]
Machine learning - Maşın öyrənməsi

**Comment by kerimovscreations** [comment]
Challenge - Məşğələ

**Comment by kerimovscreations** [comment]
Aİ - Sİ (Suni İntellekt)
ML - MÖ

**Comment by kerimovscreations** [comment]
Post-lecture quiz - Mühazirə sonrası test

**Comment by kerimovscreations** [comment]
Review & Self study - Təkrarlayın və özünüz öyrənin

**Comment by kerimovscreations** [comment]
Assignment - Tapşırıq

**Comment by kerimovscreations** [comment]
Deep Learning - Dərin öyrənmə

**Comment by kerimovscreations** [comment]
Data science - data elmi

**Reply by kerimovscreations** [reply]
Data scientist - data mühəndisi

**Comment by kerimovscreations** [comment]
Sketchnote - eskiz

**Comment by kerimovscreations** [comment]
Development - proqramlaşdırma, tərtibatçılıq, inkişaf

**Comment by kerimovscreations** [comment]
development workflow - proqramlaşdırma axını

**Comment by kerimovscreations** [comment]
Pre-lecture quiz - Mühazirədən əvvəl test

**Comment by kerimovscreations** [comment]
Click the image above for a short video working through preparing the data for this lesson. - 
Bu dərs üçün datanın hazırlanmasının üzərindən keçən qısa video üçün yuxarıdakı şəkilin üzərinə klikləyin.

**Comment by kerimovscreations** [comment]
Learning path - təlim yolu

**Comment by aliyevakhalida** [comment]
bar graph - zolaqlı diaqram

**Comment by kerimovscreations** [comment]
Classifier - qruplaşdırıcı
Clustering - klasterləşdirmə

**Reply by kerimovscreations** [reply]
@aliyevakhalida  @muradtries 
Təsnifatlandırıcı yoxsa qruplaşdırıcı? yekun qərarımız?

**Reply by muradtries** [reply]
Oxuyucu olaraq mənə qruplaşdırıcı sözü həm rahat deyilə bilən, həm də rahat başa düşülən kimi gəlir. Nəzərə alanda ki, bu kurs elmi ədəbiyyat yox hər kəs üçün nəzərdə tutulub, qruplaşdırıcı ilə davam etməyimizin tərəfdarıyam.

**Reply by kerimovscreations** [reply]
@muradtries clustering daha çox qruplaşdırmaya uyğundur məncə.

**Reply by muradtries** [reply]
@kerimovscreations yanlış başa düşməmişəmsə, eyni tərcümənin tərəfdarıyıq.

**Comment by kerimovscreations** [comment]
dataset - data massivi

**Comment by kerimovscreations** [comment]
Computational linguistics - Komputer dilçiliyi

**Comment by kerimovscreations** [comment]
Natural Language Processing - Təbii dil emalı

**Comment by kerimovscreations** [comment]
Prerequisites - İlkin şərtlər

**Reply by kerimovscreations** [reply]
İlkin tələblər*

**Comment by kerimovscreations** [comment]
Credits - Təşəkkürlər

**Comment by kerimovscreations** [comment]
Bias - tərəfkeşlik

**Reply by muradtries** [reply]
qərəzlilik, tərəfkeşlik daha uyğundur

**Comment by kerimovscreations** [comment]
Train - öyrətmə 

Misal - training set - öyrətmə dəsti

**Reply by kerimovscreations** [reply]
@muradtries sənin fikrin?

**Reply by muradtries** [reply]
öyrətmə daha uyğun görünür ml kontekstində

**Comment by kerimovscreations** [comment]
set - dəst

Misal - test sets - test dəstləri

**Comment by kerimovscreations** [comment]
Test - test

**Reply by kerimovscreations** [reply]
@muradtries sənin fikrin?

**Reply by muradtries** [reply]
test olaraq saxlamağı düşünürəm. dilimizdə də test sözü var və eyni mənanı ifadə edir

**Comment by kerimovscreations** [comment]
Reinforcement - gücləndirici

**Reply by kerimovscreations** [reply]
@muradtries bunun tərcüməsi alternativi gücləndiricidir. Lakin mən möhkəmlənin mövzuya da uyğun hesab edirəm. Səncə?

**Reply by muradtries** [reply]
Gücləndirici öyrənmə mənə daha düzgün gəlir. Çünki bu tip öyrənmədə model etdiyi səhvləri bir də təkrarlamamağa çalışaraq daha güclü olmağa çalışır

**Comment by kerimovscreations** [comment]
action - əməliyyat

**Reply by muradtries** [reply]
əgər işləndiyi kontekstlərdə uyğun görünürsə, "iş" olaraq da tərcümə oluna bilər

**Comment by kerimovscreations** [comment]
polarity - şkala

Nümunə: sentiment polarity - hiss şkalası

**Comment by kerimovscreations** [comment]
sentiment - hiss

**Comment by kerimovscreations** [comment]
root folder - ana qovluq

**Reply by muradtries** [reply]
bəlkə "ana qovluq" götürək?

**Comment by kerimovscreations** [comment]
supervised - nəzarətli
